userspace/libsinsp/sinsp.cpp

```diff
@@ -951,12 +951,12 @@ void sinsp::import_user_list()
 	{
 		for(j = 0; j < ul->nusers; j++)
 		{
-			m_usergroup_manager.add_user("", ul->users[j].uid, ul->users[j].gid, ul->users[j].name, ul->users[j].homedir, ul->users[j].shell);
+			m_usergroup_manager.add_user(sinsp_usergroup_manager::HOST_CONTAINER_ID, -1, ul->users[j].uid, ul->users[j].gid, ul->users[j].name, ul->users[j].homedir, ul->users[j].shell);
```

Comment:
I was about to ask you to document -1, but it would be nicer to have another constant actually